assignments/closure.js

```diff
@@ -1,14 +1,25 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
-
+const a = 0
+function addOne (num) {
+  const a = 1;
```

Comment:
```suggestion
  console.log(a)
```
The way you have it here you are not demonstrating closure. A closure is a concept in JS where an inner function has access to the variables declared in it's outer environment. So to demonstrate closure, you should show that a can be accessed from within addOne, even though it was declared outside of it.